server/src/immich/api-v1/asset/asset.service.ts

```diff
@@ -566,66 +529,15 @@ export class AssetService {
     return { filepath: asset.resizePath, contentType: 'image/jpeg' };
   }
 
-  private async streamFile(filepath: string, res: Res, headers: Record<string, string>, contentType?: string | null) {
-    await fs.access(filepath, constants.R_OK);
-    const { size, mtimeNs } = await fs.stat(filepath, { bigint: true });
-
+  async streamFile(res: Res, filepath: string, contentType?: string | null) {
     if (contentType) {
-      res.header('Content-Type', contentType);
+      res.contentType(contentType);
     }
-
-    const range = this.setResRange(res, headers, Number(size));
-
-    // etag
-    const etag = `W/"${size}-${mtimeNs}"`;
```

Comment:
@fyfrey I believe we need ETag header, correct?

Reply:
It sets an ETag header that seems to work, but I don't know what it uses for it.

Reply:
The mobile app uses the property for tracking the changes believe.

https://github.com/immich-app/immich/pull/1031

Reply:
Express is configured to use weak etags by default. We don't need to manually configure them like this.